### PR TITLE
testing/picotts: new aport

### PR DESCRIPTION
**Pull request by etrombly**
https://github.com/naggety/picotts
Pico TTS: text to speech voice synthesizer from SVox

### CODE REVIEW

File: testing/picotts/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+pkgname="picotts"
+pkgver="0.1"
+pkgrel=0
+pkgdesc="Pico TTS: text to speech voice synthesizer from SVox"
+url="https://github.com/naggety/picotts"
+arch="all"
+license="Apache-2.0"
+depends="popt"
+makedepends="popt-dev autoconf automake libtool"
+install=""
```
Comment (maxice8) [inline testing/picotts/APKBUILD]: empty variables can be removed

File: testing/picotts/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+subpackages="$pkgname-dev"
+source="https://github.com/naggety/picotts/archive/e3ba46009ee868911fa0b53db672a55f9cc13b1c.zip"
+builddir="$srcdir/picotts-e3ba46009ee868911fa0b53db672a55f9cc13b1c/pico"
+
+build() {
+	./autogen.sh
```
Comment (maxice8) [inline testing/picotts/APKBUILD]: have this in prepare() not build()

File: testing/picotts/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+pkgrel=0
+pkgdesc="Pico TTS: text to speech voice synthesizer from SVox"
+url="https://github.com/naggety/picotts"
+arch="all"
+license="Apache-2.0"
+depends="popt"
+makedepends="popt-dev autoconf automake libtool"
+install=""
+subpackages="$pkgname-dev"
+source="https://github.com/naggety/picotts/archive/e3ba46009ee868911fa0b53db672a55f9cc13b1c.zip"
```
Comment (maxice8) [inline testing/picotts/APKBUILD]: this is not a versioned archive

Comment (etrombly) [reply testing/picotts/APKBUILD]: The author hasn't made any releases, so I'm grabbing a specific commit. How is this normally handled?

Comment (maxice8) [reply testing/picotts/APKBUILD]: Creating an issue on the repository of the author asking for a release and then using it. If he takes too long a git commit with version `0_git$(date)` is acceptable but we really want a tagged release

File: testing/picotts/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# Contributor: Eric Trombly <etrombly@yahoo.com>
+# Maintainer:
```
Comment (Cogitri) [inline testing/picotts/APKBUILD]: Dont you want to add yourself as maintainer?

File: testing/picotts/APKBUILD (hunk 3 above)
Comment (Cogitri) [reply testing/picotts/APKBUILD]: You'll need to adjust the package version.

Comment (etrombly) [reply testing/picotts/APKBUILD]: Still no response from the author. I'll give it another day or two.

File: testing/picotts/APKBUILD (hunk 4 above)
Comment (Cogitri) [reply testing/picotts/APKBUILD]: Since you've created it it seems fair to have you as maintainer :)

**Comment by Cogitri** [comment]
Please ping me/request a review once you're added yourself as maintainer and fixed the pkgver.

**Pull request closed without merging**